src/frontend/src/views/Data/components/TreeTableNameCell/style.ts

```diff
@@ -1,39 +1,57 @@
 import styled from "@emotion/styled";
-import { getColors } from "czifui";
+import { getColors, Props } from "czifui";
 import { RowContent } from "src/common/components/library/data_table/style";
 import OpenInNewIcon from "src/common/icons/OpenInNew.svg";
 import { icon } from "../../../../common/components/library/data_table/style";
 
-export const StyledOpenInNewIcon = styled(OpenInNewIcon)`
+export interface ExtraProps extends Props {
+  disabled?: boolean;
+}
+
+const doNotForwardProps = ["disabled"];
+
+export const StyledOpenInNewIcon = styled(OpenInNewIcon, {
+  shouldForwardProp: (prop) => !doNotForwardProps.includes(prop as string),
```

Comment:
question i've had when seeing this is how does this work exactly with not forwarding the `doNotForwardProps`?

Reply:
is shouldForwardProp a special keyword for emotion/styled?

Reply:
Yes, exactly. It an optionally configurable prop to let emotion's styled know whether to forward props to the final dom element.

https://styled-components.com/docs/api#shouldforwardprop